Trakt/ScheduledTasks/SyncLibraryTask.cs

```diff
@@ -163,40 +183,43 @@ public async Task ExecuteAsync(IProgress<double> progress, CancellationToken can
                 baseQuery.Limit = Limit;
                 baseQuery.StartIndex = offset;
 
-                var libraryMovies = _libraryManager.GetItemList(baseQuery);
+                var libraryMovies = _libraryManager.GetItemList(baseQuery)
+                    .OfType<Movie>()
+                    .Where(item => !traktUser.LocationsExcluded.Any(directory => item.Path.Contains(directory, StringComparison.OrdinalIgnoreCase)))
```

Comment:
If this completely filters the query result any future items won't be processed

Reply:
What exactly do you mean? The idea is to just sync the current state of the library, deletion, watched events are handled otherwise so it shouldn't be problematic if the data changes while syncing.

Reply:
In the edge case where every item in the page is filtered out the loop will end without processing any further pages of items

Reply:
Allright, I just ran into thius problem while fixing th eother task. Should be fixed in the next (way too large) commit.